ocsci/pytest_customization/ocsci_logging.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import os
+import time
+import pytest
+
+from ocsci import config as ocsci_config
+
+FORMATTER = logging.Formatter(
+    "%(asctime)s - %(levelname)s - %(name)s.%(funcName)s.%(lineno)d"
+    " - %(message)s"
+)
+ep_time = int(time.time())
+log_dir = "/logs_" + str(ep_time)
+log_dir_path = ocsci_config.RUN['log_dir'] + log_dir
```

Comment:
Use `os.path.join(ocsci_config.RUN['log_dir'], log_dir)` instead of concat here and on the next line.

Reply:
It is better to log in the upper log_dir then in the tree structure, makes it easier to browse when things fail.

Reply:
> Use `os.path.join(ocsci_config.RUN['log_dir'], log_dir)` instead of concat here and on the next line.

Done

Reply:
> It is better to log in the upper log_dir then in the tree structure, makes it easier to browse when things fail.

Even initially I thought the same but when our test cases grows, and if 2 test cases has same test case id, then logging in the single directory create problems of overriding the logs. Even though if we append number at the end of file, its difficult for users to differentiate between 2 identical test cases. 

Traversing through directory structure of logs is easy because user knows where the test case resides.

Please share your thoughts

Reply:
Sure, Lets this one goes!